deploy/crds/apps.open-cluster-management.io_multiclusterapplicationsetreports.yaml

```
@@ -21,8 +22,12 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: MulticlusterApplicationSetReport is the Schema for the MulticlusterApplicationSetReport
-          API.
+        description: MulticlusterApplicationSetReport provides a report of the status
+          of an application from all the managed clusters where the application is
+          deployed on. It provides a summary of the number of clusters in the various
+          states. If an error or warning occurred when installing the application
+          on a managed cluster, the conditions, including the waring/error message,
```

According to IBM SG, avoid /

```suggestion
          on a managed cluster, the conditions, including the waring and error message,
```